src/components/status-check.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Box, Flex, space, themeColor, color } from '@blockstack/ui';
+import { border } from '@common/utils';
+import { Link } from '@components/mdx';
+import { Text } from '@components/typography';
+import fetch from 'isomorphic-unfetch';
+import { CheckmarkCircleIcon, ExclamationMarkCircleIcon } from '@blockstack/ui';
+
+export const StatusCheck: React.FC<any> = () => {
+  const [status, setStatus] = React.useState(false);
+  const STATUS_CHECKER_URL = 'https://status-checker-blockstack.herokuapp.com';
+
+  React.useEffect(() => {
+    fetch(`${STATUS_CHECKER_URL}/json`)
+      .then(r => r.json())
+      .then(data => {
+        if (data && data.masterNodePings.length > 1) {
+          setStatus(data.masterNodePings[0].value === 1);
+        }
+      })
+      .catch(err => console.log(err));
+  }, []);
```

Comment:
I would recommend using `useSWR` for this kind of thing: https://swr.now.sh/

Reply:
using `swr` will give us a lot nice things out of the box (from their site):
```
Fast page navigation
Polling on interval
Data dependency
Revalidation on focus
Revalidation on network recovery
```

Reply:
nice!